### PR TITLE
Add clientEnvironment to CNHi documentation

### PR DESCRIPTION
**Pull request by gabrieltron**

### CODE REVIEW

**Comment by gabrieltron** [comment]
[[Documentation] Add "production" to CNHi request body](https://trello.com/c/RkyjRqb4/1476-documentation-add-production-to-cnhi-request-body)

File: docs/user_management_endpoints.md [hunk 1]
```diff
@@ -2031,7 +2032,7 @@ A "CNHI Credentials" resource as a JSON.
   const axios = require('axios')
   const TOKEN = 'YOUR_TOKEN'
 
-  const endpoint = 'https://api.withleaf.io/services/usermanagement/api/cnhi-credentials/{id}'
+  const endpoint = 'https://api.withleaf.io/services/usermanagement/api/cn-hi-credentials/{id}'
```
Comment (fabyan-alexander) [inline docs/user_management_endpoints.md:2035]: wow 😓

File: docs/user_management_endpoints.md [hunk 2]
```diff
@@ -2387,7 +2395,8 @@ A CNHI Credentials with status.
   ```shell
   curl -X POST \
       -H 'Authorization: Bearer YOUR_TOKEN' \
-      -d '{"clientId": "str","clientSecret": "str","refreshToken": "str"}' \
+      -H 'Content-Type: application/json' \
+      -d '{"clientId": "str", "clientSecret": "str", "refreshToken": "str", "clientEnvironment": "STAGE or PRODUCTION"}' \
       'https://api.withleaf.io/services/usermanagement/api/users/{leafUserId}/cnhi-credentials'
```
Comment (fabyan-alexander) [inline docs/user_management_endpoints.md:2400]:
```suggestion
      'https://api.withleaf.io/services/usermanagement/api/users/{leafUserId}/cn-hi-credentials'
```

**Review by fabyan-alexander — APPROVED** [review]
Nice work! Just a single extra fix

File: docs/user_management_endpoints.md (hunk 2 above)
Comment (gabrieltron) [reply docs/user_management_endpoints.md:2400]: sadly, that is correct

**Pull request merged**